src/main/minio.js

```diff
@@ -354,7 +355,9 @@ export class Client {
     }
     if (!options.headers) options.headers = {}
     if (options.method === 'POST' || options.method === 'PUT' || options.method === 'DELETE') {
-      options.headers['content-length'] = payload.length
+      const encoder = new util.TextEncoder()
+      const buffer = encoder.encode(payload)
```

Comment:
a payload can be really large this cannot work when there are large payloads @loremaps